app/assets/stylesheets/components/_publisher-metadata-with-logo.scss

```diff
@@ -0,0 +1,24 @@
+@import "govuk_publishing_components/individual_component_support";
```

Comment:
I think we might have to find somewhere different for this file - if something's a component, it has to meet a set of criteria (it has to have a specific set of component tests, docs, the class names for stylesheet matching have to have an app-c- prefix, etc). This wasn't a component in government-frontend, so perhaps we can find a better place for it in frontend. (maybe we can just have a stylesheets/views/shared folder?)